Userland/Libraries/LibVideo/CodecID.h

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2023, Stephan Vedder <stephan.vedder@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Format.h>
+
+namespace Video {
+
+enum class CodecID : u32 {
+    Unknown,
+    // On2 / Google
+    VP8,
+    VP9,
+    // MPEG
+    H261,
+    MPEG1,
+    H262,
+    MPEG2 = H262,
+    H263,
+    AVC = H263,
+    H264,
+    H265,
+    HEVC = H265,
+    // AOMedia
+    AV1,
+    // Xiph
+    Theora,
+    Vorbis,
+    Opus,
+};
+
+}
+
+namespace AK {
+template<>
+struct Formatter<Video::CodecID> : Formatter<StringView> {
+    ErrorOr<void> format(FormatBuilder& builder, Video::CodecID value)
+    {
+        StringView codec;
+        switch (value) {
+        case Video::CodecID::Unknown:
+            codec = "Unknown"sv;
+            break;
+        case Video::CodecID::VP8:
+            codec = "VP8"sv;
+            break;
+        case Video::CodecID::VP9:
+            codec = "VP9"sv;
+            break;
+        case Video::CodecID::H261:
+            codec = "H261"sv;
+            break;
+        case Video::CodecID::H262:
+            codec = "H262"sv;
+            break;
+        case Video::CodecID::H263:
+            codec = "H263"sv;
+            break;
+        case Video::CodecID::H264:
+            codec = "H264"sv;
+            break;
+        case Video::CodecID::H265:
+            codec = "H265"sv;
+            break;
```

Comment:
These could include the `.`, but I guess it's a question of whether we want the formatter to print the human-readable name in general or specifically the name of the enum constant.

Reply:
I would agree that it should include the `.` as well.

Reply:
> These could include the `.`, but I guess it's a question of whether we want the formatter to print the human-readable name in general or specifically the name of the enum constant.

Tried to reply to this and made a separate review comment instead, whoops. The formatter should definitely use human-friendly names for these, so it should include the ".", if we're going to use the ITU-T names.

Reply:
H.264